src/plugins/slack-now-playing/main.ts

```diff
@@ -368,9 +362,9 @@ async function uploadEmojiToSlack(songInfo: SongInfo, config: SlackNowPlayingCon
       // If we got here, the request was successful
       // Emoji uploaded successfully
       return true;
-    } catch (apiError) {
+    } catch (apiError: any) {
```

Comment:
instead of `any` use `unknown`, that's your intended use-case either way, since you are trying to narrow down the type later

Reply:
Refactored to `unknown` in a8350ecfaa0d9caff07db2ed062ff50e0de97707.